readme.md

```diff
@@ -86,12 +98,12 @@ You can also access the Application Link details page by going to **Jenkins** >
 
 #### Part 2 - Create an Application Link to Jenkins
```

Comment:
Can we update this to `## 2. Create an Application Link to Jenkins` to be consistent with the other numbering?

Reply:
That's fair. @ali3-atlassian can you please remove 'part' from both. I can't add a suggestion sorry.